docs/source/amp.rst

```diff
@@ -1,218 +1,22 @@
 .. role:: hidden
     :class: hidden-section
 
-Automatic Mixed Precision package - torch.cuda.amp
+Automatic Mixed Precision package - torch.autocast
 ==================================================
 
-.. automodule:: torch.cuda.amp
-.. currentmodule:: torch.cuda.amp
-
-:class:`torch.cuda.amp` and :class:`torch` provide convenience methods for mixed precision,
+:class:`torch.autocast` provides convenience methods for mixed precision,
 where some operations use the ``torch.float32`` (``float``) datatype and other operations
-use ``torch.float16`` (``half``). Some ops, like linear layers and convolutions,
-are much faster in ``float16``. Other ops, like reductions, often require the dynamic
+use lower precision floating point datatype (``lower_precision_fp``): ``torch.float16`` (``half``) or ``torch.bfloat16``. Some ops, like linear layers and convolutions,
+are much faster in ``lower_precision_fp``. Other ops, like reductions, often require the dynamic
 range of ``float32``.  Mixed precision tries to match each op to its appropriate datatype.
 
-Ordinarily, "automatic mixed precision training" uses :class:`torch.autocast` and
-:class:`torch.cuda.amp.GradScaler` together, as shown in the :ref:`Automatic Mixed Precision examples<amp-examples>`
-and `Automatic Mixed Precision recipe <https://pytorch.org/tutorials/recipes/recipes/amp_recipe.html>`_.
-However, :class:`torch.autocast` and :class:`GradScaler` are modular, and may be used separately if desired.
+For CUDA and CPU, :doc:`cuda.amp` and :doc:`cpu.amp` are also provided seperately. ``torch.autocast("cuda", args...)`` is equivalent to ``torch.cuda.amp.autocast(args...)``. ``torch.autocast("cpu", args...)`` is equivalent to ``torch.cpu.amp.autocast(args...)``.
```

Comment:
It seems bad to drop the link to the recipe, the recipe seems like it should still be useful.

Reply:
Thanks for the suggestions.  Added the link of `GradScaler` back.